### PR TITLE
Readme: mention `Strings::matchAll`

### PR DESCRIPTION
**Pull request by janedbal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ This extension provides following features:
 * Magic [Nette\Object and Nette\SmartObject](https://doc.nette.org/en/2.4/php-language-enhancements) properties
 * Event listeners through the `on*` properties
 * Defines early terminating method calls for Presenter methods to prevent `Undefined variable` errors
+* Understand the exact array shape coming from `Nette\Utils\Strings::matchAll()` based on pattern
```
Comment (ondrejmirtes) [inline README.md]: Please mention match() too

**Review by ondrejmirtes — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (janedbal) [reply README.md]: Adjusted, is this sufficient?

Comment (ondrejmirtes) [reply README.md]: No, mention the separately ;) What if Nette adds another method with this pattern and we don't support it? ;)

Comment (janedbal) [reply README.md]: Done

**Comment by ondrejmirtes** [comment]
Thank you!

**Pull request merged**